changes/pr4499.yaml

```diff
@@ -0,0 +1,2 @@
+feature:
+  - "Key Value Store - [#4499](https://github.com/PrefectHQ/prefect/pull/4499)"
```

Comment:
Could this be more user-facing?

Reply:
updated, I'm not a very good marketer though 🤷‍♂️

Reply:
Maybe "Add interface for backend key-value metadata store" @zangell44

Reply:
fine by me, i like it